icons/gtk.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>GTK</title><path d="M13.695 0v8.065l8.93-2.748zm-.325.09L1.415 3.372 9.423 9.38l3.946-1.215zM.76 4.112l.494 12.366 7.502-2.22v-4.148zm22.482 1.933-9.546 2.938v3.945l8.52 5.938zM13.37 9.084l-3.475 1.07v3.768l3.475-1.03zm.134 4.108-3.61 1.068V24l11.965-4.984zm-4.748 1.405-7.488 2.217 7.488 6.99z"/></svg>
```

Comment:
When I derive the icon from the Wikipedia logo, I see a notable difference between yours (blue) and mine (red):

![gtk-comparison](https://user-images.githubusercontent.com/2206639/167151231-fd120918-ac2e-4941-a9a4-eed6222e3936.svg)

I converted the strokes to paths, disabled the strokes on the original shapes, and subtracted the stroke paths from the original shapes. Then, I created a union of the paths, resized them, and centered the path.

Could you please double-check?

```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>GTK</title><path d="M9.01 23.773V14.45l-7.584 2.245Zm0-13.87L.91 3.828l.502 12.526 7.597-2.249ZM9.57 24l12.353-5.146-8.285-5.775-4.068 1.204ZM23.09 5.815l-9.257 2.849v4.148l8.237 5.741ZM9.57 9.975v3.964l3.932-1.164v-4.01Zm-.228-.52 4.16-1.28V0L1.231 3.37ZM22.715 5.34 13.833.052v8.021Z"/></svg>
```